.github/workflows/tests.yml

```diff
@@ -1,7 +1,5 @@
 on:
   push:
-    branches: [ main ]
-  pull_request:
```

Comment:
do you want to keep that or is it just for you CI tests?

Reply:
yes i want to keep! 😄 
the previous behaviour is to execute test & lint only at PR creation or on push on main.
But i think its better to execute it at every push on every branch